frappe/core/doctype/user/user.py

```diff
@@ -728,8 +730,14 @@ def _get_user_for_update_password(key, old_password):
 	# verify old password
 	result = frappe._dict()
 	if key:
-		result.user = frappe.db.get_value("User", {"reset_password_key": key})
-		if not result.user:
+		user = frappe.db.get_value("User", {"reset_password_key": key}, ["name", "reset_password_key_datetime"])
+		result.user, res_pass_key_datetime = user if user else (None, None)
+
+		if result.user:
+			res_pass_link_exp_sec = frappe.db.get_single_value("System Settings", "reset_password_link_expiry_seconds")
```

Comment:
`password_reset_expiry` - avoid short-forms

Reply:
changed it to `reset_password_link_expiry`